testnet/stacks-node/src/nakamoto_node/relayer.rs

```diff
@@ -99,6 +98,91 @@ impl fmt::Display for RelayerDirective {
     }
 }
 
+/// Last commitment data
+/// This represents the tenure that the last-sent block-commit committed to.
+pub struct LastCommit {
+    /// block-commit sent
+    block_commit: LeaderBlockCommitOp,
+    /// the sortition tip at the time the block-commit was sent
+    burn_tip: BlockSnapshot,
+    /// the stacks tip at the time the block-commit was sent
+    stacks_tip: StacksBlockId,
+    /// the tenure consensus hash for the tip's tenure
+    tenure_consensus_hash: ConsensusHash,
+    /// the start-block hash of the tip's tenure
+    start_block_hash: BlockHeaderHash,
+    /// What is the epoch in which this was sent?
+    epoch_id: StacksEpochId,
+    /// commit txid (to be filled in on submission)
+    txid: Txid,
```

Comment:
Should be an `Option`, `[0; 32]` is a bad `None`.

Reply:
Sure